bundles/org.eclipse.e4.ui.workbench/META-INF/MANIFEST.MF

```diff
@@ -33,7 +33,8 @@ Export-Package: org.eclipse.e4.ui.internal.workbench;
  org.eclipse.e4.ui.internal.workbench.handlers;x-internal:=true,
  org.eclipse.e4.ui.workbench,
  org.eclipse.e4.ui.workbench.lifecycle,
- org.eclipse.e4.ui.workbench.modeling
+ org.eclipse.e4.ui.workbench.modeling,
+ org.eclipse.ui.activitysupport
```

Comment:
Please use common naming schema for packages here. `org.eclipse.ui.e4.activitysupport`